### PR TITLE
docs: note args --> config migration

### PR DESCRIPTION
**Pull request by Aghassi**
This ensures that folks know that `config` is the replacement for `args` when coming from `@bazel/esbuild`. This is an issue I hit, see this discussion in slack https://bazelbuild.slack.com/archives/CEZUUKQ6P/p1675462695407799?thread_ts=1674525810.470589&cid=CEZUUKQ6P

CC @mattem @gregmagolan

### CODE REVIEW

File: esbuild/private/esbuild.bzl [hunk 1]
```diff
@@ -159,7 +159,7 @@ See https://esbuild.github.io/api/#target for more details
     "config": attr.label(
         mandatory = False,
         allow_single_file = True,
-        doc = """Configuration file used for esbuild. Note that options set in this file may get overwritten.
+        doc = """Configuration file used for esbuild. Note that options set in this file may get overwritten. If you formally used `args` from `@bazel/esbuild`, replace it with this attribute.
```
Comment (alexeagle) [inline esbuild/private/esbuild.bzl:162]:
```suggestion
        doc = """Configuration file used for esbuild. Note that options set in this file may get overwritten. If you formerly used `args` from rules_nodejs' npm package `@bazel/esbuild`, replace it with this attribute.
```

**Comment by alexeagle** [comment]
I think it would be better just to make a separate docs/migration.md with whatever table of replace-this-with-that or other notes you've got.

**Comment by Aghassi** [comment]
> I think it would be better just to make a separate docs/migration.md with whatever table of replace-this-with-that or other notes you've got.

This is the only arg I have to replace currently it seems, but I'm happy to do that too. I probably need to do this for each of the rules. Would you prefer that?

**Review by alexeagle — APPROVED** [review]

**Comment by alexeagle** [comment]
just merging to cleanup PRs

**Pull request merged**